stdlib/public/core/UnsafeRawPointer.swift

```diff
@@ -434,6 +453,10 @@ public struct UnsafeRawPointer: _Pointer {
 #endif
   }
 
+  // FIXME(NCG): Add a consuming analogue of `load`, like `move(fromByteOffset:as:_:)`
```

Comment:
Wouldn't this imply a mutation of the underlying bytes, through the non-mutable pointer?

Reply:
Yep, we'd only want this for UnsafeMutableRawPointer